main.c

```diff
@@ -73,6 +73,17 @@ int main(void) {
     TRISC1 = 1;
     ANSELC1 = 0;
     CANRXPPS = 0x11; // make CAN read from C1 (page 264-265)
+    
+    #if (BOARD_UNIQUE_ID == BOARD_ID_CHARGING_AIRBRAKE || BOARD_UNIQUE_ID == BOARD_ID_CHARGING_PAYLOAD)
+    pwm_init();
+    #endif
+
+    #if (BOARD_UNIQUE_ID == BOARD_ID_CHARGING_AIRBRAKE)
+    //suspicious lmao
```

Comment:
is it actually sussy still?

Reply:
well we tested it and it works so not sussy anymore?

Reply:
This and the other notes about code comments were mostly rhetorical. Probably could have communicated that better. Anyway, this and all the other comments that are out of date/not relevant should be removed.